packages/api/__tests__/utility.ts

```diff
@@ -37,6 +39,13 @@ class ExampleURLAdapter implements URLAdapter {
   getArticlePreviewURL(token: string): string {
     return `https://demo.wepulish.ch/article/preview/${token}`
   }
+
+  getCommentURL(item: PublicArticle | PublicPage, comment: PublicComment): string {
+    if (comment.itemType === CommentItemType.Article) {
+      return `https://editor.demo.wepublish.media/comments/a/${item.id}/${item.slug}/${comment.id}`
```

Comment:
I know this is only a test but the URL would still not be on the editor. It would be more something like `https://demo.wepublish.media/...`